src/main/java/roomescape/jwt/JwtTokenProvider.java

```diff
@@ -0,0 +1,42 @@
+package roomescape.jwt;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.SecretKey;
+import java.util.Date;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Component
+public class JwtTokenProvider {
+
+    private SecretKey secretKey;
+    private long validityInMilliseconds;
+
+    public JwtTokenProvider(@Value("${security.jwt.token.secret-key}") String secretKey,
+                            @Value("${security.jwt.token.expire}") long validityInMilliseconds) {
+        this.secretKey = Keys.hmacShaKeyFor(secretKey.getBytes(UTF_8));
+        this.validityInMilliseconds = validityInMilliseconds;
+    }
+
+    public String createToken(String email) {
+        return Jwts.builder()
+                .claim("email", email)
+                .issuedAt(new Date(System.currentTimeMillis()))
+                .expiration(new Date(System.currentTimeMillis() + validityInMilliseconds))
```

Comment:
추가로 현재에 대한 값을 위의 변수로 선언해서 사용해도 좋을 것 같아요!
각각 메서드에서 new Date 값을 구하면 실행 시점에 따라 값이 변경될 우려가 있겠네요!
```suggestion
                .issuedAt(now)
                .expiration(now + validityInMilliseconds))
```